src/CMakeLists.txt

```diff
@@ -4,9 +4,12 @@ project(chia-log-analysis VERSION 0.5)
 
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED True)
+set(CMAKE_CXX_FLAGS "-lstdc++fs")  
+
 
 configure_file(CMakeConfig.h.in CMakeConfig.h)
 
 add_executable(chia-log-analysis main.cpp InputHandler.cpp TimeAndDate.cpp)
 
-target_include_directories(chia-log-analysis PUBLIC "${PROJECT_BINARY_DIR}")
\ No newline at end of file
+target_include_directories(chia-log-analysis PUBLIC "${PROJECT_BINARY_DIR}")
+target_include_libraries(chia-log-analysis PUBLIC -lstdc++fs)
```

Comment:
When running `cmake -S src -B build` with this line I get the following error under my environments (msvc 16 & gcc 10, cmake 3.20):
```
CMake Error at CMakeLists.txt:15 (target_include_libraries):
  Unknown CMake command "target_include_libraries".
```